python/docs/source/reference/timestream/misc.md

```diff
@@ -12,6 +12,7 @@
     Timestream.filter
     Timestream.hash
     Timestream.if_
+    Timestream.is_valid
```

Comment:
`is_valid` was added as `is_not_null` in `logical.md`. I don't think we need to add it.